ste/mgr-JobMgr.go

```diff
@@ -97,14 +97,16 @@ type IJobMgr interface {
 	AddSuccessfulBytesInActiveFiles(n int64)
 	SuccessfulBytesInActiveFiles() uint64
 	CancelPauseJobOrder(desiredJobStatus common.JobStatus) common.CancelPauseResumeResponse
+
+	ChangeLogLevel(pipeline.LogLevel) bool
 }
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 func NewJobMgr(concurrency ConcurrencySettings, jobID common.JobID, appCtx context.Context, cpuMon common.CPUMonitor, level common.LogLevel,
-	       commandString string, logFileFolder string, tuner ConcurrencyTuner,
-	       pacer PacerAdmin, slicePool common.ByteSlicePooler, cacheLimiter common.CacheLimiter, fileCountLimiter common.CacheLimiter,
-	       jobLogger common.ILoggerResetable) IJobMgr {
+	commandString string, logFileFolder string, tuner ConcurrencyTuner,
+	pacer PacerAdmin, slicePool common.ByteSlicePooler, cacheLimiter common.CacheLimiter, fileCountLimiter common.CacheLimiter,
+	jobLogger common.ILoggerResetable) IJobMgr {
```

Comment:
Undo unwanted changes. 
There are few more blank space changes.

Reply:
Its gofmt, change whenever i save on IDE it moved it.